rebar.config

```diff
@@ -39,7 +39,8 @@
             , {preprocess, true}
             ]}.
 
-{deps, [{eredis, {git, "git://github.com/Nordix/eredis.git", {ref, "a83413a"}}},
+{deps, [{eredis, {git, "git://github.com/Nordix/eredis.git",
+                  {tag, "v1.3.0-nordix"}}},
```

Comment:
Shouldn't be `v1.3.1-nordix` ?

Reply:
Yes, you're right. Should we wait a bit until it's actually official?

Reply:
Now we should depend on 1.3.2 on hex 😅